src/main/kotlin/blackjack/model/GameResult.kt

```diff
@@ -46,6 +46,16 @@ class GameResult {
         dealerResults[dealerResult] = dealerResults.getOrDefault(dealerResult, DEFAULT_COUNT) + ADD_RESULT_COUNT
     }
 
+    fun getParticipantProfitResult(): List<Double> {
+        val playersProfit = mutableListOf<Double>()
+        getPlayerResults().forEach { player, result ->
+            val earningRate = player.calculateEarningRate(result, player.getBlackJackState())
+            val profit = player.calculateProfit(player, earningRate)
+            playersProfit.add(profit)
+        }
+        return playersProfit
+    }
```

Comment:
좋습니다.
다만 아래와 같이
```suggestion
    fun getParticipantProfitResult(): List<Double> {
        val result = players.getProfitResult()
        return result
    }
```

와 같이 일급컬렉션을 사용하여 객체에게 도메인 로직을 집중시킬 수 있습니다.
이후 오목 미션 진행하시면서도 객체가 할 수 있는 일을 최대한 위임하는 연습을 하시면 좋을 것 같습니다 👍